grid/address/geocoding/maps/locationiq.suma

```diff
@@ -88,14 +88,15 @@ map ReverseGeocode {
       }
 
       // region
-      set if (body.address.city_district) {
-        mappedAddress.addressRegion = body.address.city_district;
-      }
+      addressRegion = body.address.city_district || ''
       set if (body.address.county) {
-        mappedAddress.addressRegion = `${mappedAddress.addressRegion}, ${body.address.county}`;
+        addressRegion = addressRegion ? `${addressRegion}, ${body.address.county}` : body.address.county;
       }
       set if (body.address.state) {
-        mappedAddress.addressRegion = `${mappedAddress.addressRegion}, ${body.address.state}`
+        addressRegion = addressRegion ? `${addressRegion}, ${body.address.state}` : body.address.state;
```

Comment:
BTW, I think this could be more elegant by using an array, but let's keep it now.